### PR TITLE
docs: 3.1 release branch

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
This PR contains the documentation changes for the 3.1 release.

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1058#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04-clusters/01-public-cloud/01-aws/02-architecture.md
Comment (resmaa) [inline content/docs/04-clusters/01-public-cloud/01-aws/02-architecture.md:29]: `An Internet Gateway (IG) is created for each Virtual Private Cloud (VPC) to allow Secure Shell Protocol (SSH) access to the bastion node for debugging purposes. SSH into Kubernetes nodes is only available through the bastion node. In addition, a bastion node helps to provide access to the Amazon Elastic Compute Cloud (EC2) instances. This is because the EC2 instances are created in a private subnet, and the bastion node operates as a secure, single point of entry into the infrastructure. The bastion node can be accessed via SSH or Remote Desktop (RDP).
		`

Comment (resmaa) [inline content/docs/04-clusters/01-public-cloud/01-aws/02-architecture.md:29]: The bastion node can be accessed via SSH or Remote Desktop (RDP).

i guess this sentence is in passive voice.

File: content/docs/04-clusters/01-public-cloud/01-aws/02-architecture.md
Comment (resmaa) [inline content/docs/04-clusters/01-public-cloud/01-aws/02-architecture.md:39]: `Palette also supports deploying and managing AWS Elastic Kubernetes Service (EKS) clusters. Review the architectural highlights on EKS when managed by Palette.`

File: content/docs/04-clusters/01-public-cloud/01-aws/02-architecture.md
Comment (resmaa) [inline content/docs/04-clusters/01-public-cloud/01-aws/02-architecture.md:41]: - Cluster resources such as Virtual Machines (VMs) can be provisioned into existing infrastructure (gateways, VPCs, subnets etc.) as part of static provisioning and new dedicated infrastructure as part of dynamic provisioning.

File: content/docs/04-clusters/01-public-cloud/01-aws/02-architecture.md
Comment (resmaa) [inline content/docs/04-clusters/01-public-cloud/01-aws/02-architecture.md:87]: `By default, worker pools are configured to use on-demand instances. However, you can specify spot instances when creating a cluster to take advantage of discounted spot instance pricing. The **On-Spot** option can be selected in the node configuration page during cluster creation. This option allows you to specify a maximum bid price for the nodes as a percentage of the on-demand price. The palette tracks the current price for spot instances and launches nodes when the spot price falls in the specified range.`

Comment (resmaa) [inline content/docs/04-clusters/01-public-cloud/01-aws/02-architecture.md:87]: i guess "The **On-Spot** option can be selected in the node configuration page during cluster creation. "
 is in passive voice

File: content/docs/04-clusters/01-public-cloud/01-aws/05-add-aws-accounts.md
Comment (resmaa) [inline content/docs/04-clusters/01-public-cloud/01-aws/05-add-aws-accounts.md:39]: may be we can link the permissions page here

File: content/docs/04-clusters/01-public-cloud/01-aws/08-create-cluster.md
Comment (resmaa) [inline content/docs/04-clusters/01-public-cloud/01-aws/08-create-cluster.md:17]: `Palette supports creating and managing Kubernetes clusters deployed to an AWS account. This section guides you on how to create a Kubernetes cluster in AWS that Palette manages.`

File: content/docs/04-clusters/01-public-cloud/01-aws/08-create-cluster.md
Comment (resmaa) [inline content/docs/04-clusters/01-public-cloud/01-aws/08-create-cluster.md:72]: `Populate the wizard page with the following information: name, description, tags, and AWS account. Tags on a cluster are propagated to the VMs deployed on the cloud/data center environments. Click on **Next** after completing all the required information.`

File: content/docs/04-clusters/01-public-cloud/01-aws/08-create-cluster.md
Comment (resmaa) [inline content/docs/04-clusters/01-public-cloud/01-aws/08-create-cluster.md:101]: For example, the default worker pool may be configured with the m3.large instance types for general-purpose workloads, and another worker pool with instance type g2.2xlarge can be configured to run GPU workloads.

i guess this is in passive voice.

File: content/docs/04-clusters/01-public-cloud/01-aws/08-create-cluster.md
Comment (resmaa) [inline content/docs/04-clusters/01-public-cloud/01-aws/08-create-cluster.md:137]: `Reviewing the cluster details page, you can validate that your cluster is up and running. Navigate to the left **Main Menu** and click on **Clusters**. The **Clusters** page contains a list of all available clusters managed by Palette. Click on the row for the cluster you wish to review its details page. Ensure the **Cluster Status** field contains the value **Running**.
		`

File: content/docs/04-clusters/01-public-cloud/01-aws/08-create-cluster.md
Comment (resmaa) [inline content/docs/04-clusters/01-public-cloud/01-aws/08-create-cluster.md:168]: `If a cluster is stuck in the **Deletion** state for a minimum of 15 minutes, it becomes eligible for force deletion. You can force delete a cluster from the tenant and project admin scope.`

File: content/docs/04-clusters/01-public-cloud/01-aws/08-create-cluster.md
Comment (resmaa) [inline content/docs/04-clusters/01-public-cloud/01-aws/08-create-cluster.md:169]: `To force delete a cluster, follow the steps outlined in [Deleting an AWS IaaS Cluster](#deleting-an-aws-iaas-cluster). However, after 15 minutes, a **Force Delete Cluster** option is available in the **Settings** drop-down menu. The **Settings** drop-down menu will also provide an estimated time left before the force deletion becomes available.`

File: content/docs/04-clusters/01-public-cloud/01-aws/09-eks.md
Comment (resmaa) [inline content/docs/04-clusters/01-public-cloud/01-aws/09-eks.md:18]: `
Palette supports creating and managing AWS Elastic Kubernetes Service (EKS) clusters deployed to an AWS account. This section guides you on how to create an AWS EKS cluster in AWS that Palette manages.`

File: content/docs/04-clusters/01-public-cloud/03-azure/00-architecture.md
Comment (resmaa) [inline content/docs/04-clusters/01-public-cloud/03-azure/00-architecture.md:55]: All custom storage accounts and containers will be listed in the **Cluster config** page during the cluster creation process.

i guess the above sentence is in passive voice

File: content/docs/14-troubleshooting/06-nodes.md
Comment (resmaa) [inline content/docs/14-troubleshooting/06-nodes.md:48]: `## Scenario - EKS Cluster Worker Pool Failures`

File: content/docs/14-troubleshooting/06-nodes.md
Comment (resmaa) [inline content/docs/14-troubleshooting/06-nodes.md:50]: `If your EKS cluster worker pool ends up in the `Failed` or `Create Failed` or `Error nodes failed to join` state, please refer to this [Amazon provided Runbook](https://docs.aws.amazon.com/systems-manager-automation-runbooks/latest/userguide/automation-awssupport-troubleshooteksworkernode.html`

File: content/docs/06-integrations/00-minio-operator.md
Comment (resmaa) [inline content/docs/06-integrations/00-minio-operator.md:21]: `MinIO is a Kubernetes-native high-performance object store with an Amazon S3-compatible API. Use MinIO to build high-performance infrastructure for machine learning, analytics, and application data workloads.`

**Review by resmaa — APPROVED** [review]
Added a few review comments. Please feel free to take your call on these comments @karl-cardenas-coding

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1058#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1058#pullrequestreview-1229711794" title="Approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by ritawatson — APPROVED** [review]
A few suggestions. Github burped in the middle of my adding a comment, so I didn't quite finish.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1058#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1058#pullrequestreview-1229711794" title="Approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1058#pullrequestreview-1230937895" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04-clusters/01-public-cloud/01-aws/08-create-cluster.md
Comment (ritawatson) [inline content/docs/04-clusters/01-public-cloud/01-aws/08-create-cluster.md:25]:
```suggestion
- You create an infrastructure cluster profile. Review the [Create Cluster Profiles](/cluster-profiles/task-define-profile) for guidance.
```

Comment (ritawatson) [inline content/docs/04-clusters/01-public-cloud/01-aws/08-create-cluster.md:25]:
```suggestion
- You create an infrastructure cluster profile. Review the [Create Cluster Profiles](/cluster-profiles/task-define-profile) for guidance.
```

File: content/docs/04-clusters/01-public-cloud/01-aws/08-create-cluster.md
Comment (ritawatson) [inline content/docs/04-clusters/01-public-cloud/01-aws/08-create-cluster.md:26]:
```suggestion
- There is an [EC2 Key Pair](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html) in the target region.
```

File: content/docs/04-clusters/01-public-cloud/01-aws/08-create-cluster.md
Comment (ritawatson) [inline content/docs/04-clusters/01-public-cloud/01-aws/08-create-cluster.md:27]:
```suggestion
- Palette creates compute, network, and storage resources in AWS during Kubernetes cluster provisioning. Ensure there is sufficient capacity in the AWS region to create these resources:
```

File: content/docs/04-clusters/01-public-cloud/01-aws/08-create-cluster.md
Comment (ritawatson) [inline content/docs/04-clusters/01-public-cloud/01-aws/08-create-cluster.md:39]:
```suggestion
To enable automatic subnet discovery for integration with AWS load balancer service, you should add the following tags to the virtual private network (VPC) public subnets. Replace the value `yourClusterName` with your cluster's name.
```

File: content/docs/04-clusters/01-public-cloud/01-aws/08-create-cluster.md
Comment (ritawatson) [inline content/docs/04-clusters/01-public-cloud/01-aws/08-create-cluster.md:60]:
```suggestion
2. From the **Main Menu**,  click on **Clusters**
```

**Review by ritawatson — APPROVED** [review]
with comments

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1058#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1058#pullrequestreview-1229711794" title="Approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1058#pullrequestreview-1230979686" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
